### PR TITLE
Prep and req changes

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Clean up req, and move most of the usefull free stuff in crates near alamo, ready to be deployed. 

Medical and engineer crates are ID locked, and in their respective departmens

Sulaco 

image

POS

image

## Why It's Good For The Game

The aim of the PR is to make prep shorter. This PR only will not be enough, but it's a necessary step toward it. 
All the stuff in the crate is either widely available in vendors for free, or just laying around in random location of the ship

## Changelog
:cl:
tweak: Ammo and grenade boxes are now in hangard instead of req vendor
tweak: Water/flare/mre crates are now in hangar instead of req
tweak: All items from engie vendor (sentries, heavy smartgun, mortar kit) are now in hangar
tweak: No more useless empty crates in req
tweak: The mats from synth essential kit and armory is moved in a crate in enginerring. This crate also contains the plasma cutter and cells from req, and the free phoron that was laying around. The crate is ID locked
add: One crate for field surgery added in medbay. ID locked
add: Polyhexanide pill bottle, two inside a medical crate
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -41. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by SpaceLoveSs13** [comment]
Why move ammo and weapons crate? I don't think it's worth the time to go to hanger grab ammo crate, bring it back, load ammo boxes, and put them back on alamo. It's just filling up the same time moving mre/flare/las charger would.

**Comment by Swagile** [comment]
I think the bluespace pad is the only thing I can think of at the top of my head that isn't really good.

The reason being in pool meta, its better NOT to buy a ASRS pad roundstart and buy 4 fultons (20 points total), and only buy a ASRS pad if you KNOW marines have a dedicated tadpole team or a decent engi team that will secure a FOB miner.

**Comment by Chaznoodles** [comment]
Powercreep

**Comment by SplinterGP** [comment]
Engineer stuff should be in engineer Prep, no reason to put this in there for any unga to get, we already saw engineers steal resources from all other engineers before and this lets ANYONE

**Comment by SplinterGP** [comment]
also atomize

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -114,13 +114,8 @@ GLOBAL_LIST_EMPTY(randomized_pill_icons)
...
 
-	for(var/i in 1 to 21)
-		GLOB.randomized_pill_icons += "pill[i]"
-	shuffle(GLOB.randomized_pill_icons)
-
-	shuffle(GLOB.fruit_icon_states)
-	shuffle(GLOB.reagent_effects)
-
+	for(var/i in 1 to 22)
+		GLOB.pill_icons += "pill[i]"
```
Comment (SplinterGP) [inline code/__HELPERS/global_lists.dm:118]: undocumented changes, and atomize

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -214,6 +270,52 @@
...
+	new /obj/item/storage/pill_bottle/dylovene(src)
+	new /obj/item/storage/pill_bottle/inaprovaline(src)
+	new /obj/item/storage/pill_bottle/dexalin(src)
+	new /obj/item/storage/pill_bottle/spaceacillin(src)
+	new /obj/item/storage/pill_bottle/spaceacillin(src)
+	new /obj/item/storage/pill_bottle/hypervene(src)
+	new /obj/item/storage/pill_bottle/polyhexanide(src)
+	new /obj/item/storage/pill_bottle/polyhexanide(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/quickclotplus(src)
```
Comment (SplinterGP) [inline code/game/objects/structures/crates_lockers/crates.dm:292]: why should this be in hangar ? that's medical officer gear

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 3]
```diff
@@ -214,6 +270,52 @@
...
+	new /obj/item/storage/briefcase/inflatable(src)
+	new /obj/item/healthanalyzer(src)
+	new	/obj/item/clothing/mask/surgical(src)
+	new	/obj/item/clothing/gloves/latex(src)
+	new	/obj/item/bodybag(src)
+	new	/obj/item/bodybag(src)
+	new	/obj/item/bodybag(src)
+	new /obj/item/bodybag/cryobag(src)
+	new /obj/item/bodybag/cryobag(src)
+	new /obj/item/bodybag/cryobag(src)
```
Comment (SplinterGP) [inline code/game/objects/structures/crates_lockers/crates.dm:317]: why should ANY marine have this ? this is medical/corpsman stuff.

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/items/reagent_containers/pill.dm [hunk 4]
```diff
@@ -241,3 +241,8 @@
 	//pill_desc = "An ultrazine pill. A highly-potent, long-lasting combination CNS and muscle stimulant. Extremely addictive."
 	list_reagents = list(/datum/reagent/medicine/ultrazine = 5)
 	pill_id = 21
+
+/obj/item/reagent_containers/pill/polyhexanide
+	pill_desc = "A polyhexanide pill. A sterilizing agent designed for internal use. Powerful, but dangerous. Can induce sleep"
+	list_reagents = list(/datum/reagent/medicine/polyhexanide = 15)
+	pill_id = 22
```
Comment (SplinterGP) [inline code/game/objects/items/reagent_containers/pill.dm:248]: atomize this

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by BraveMole** [comment]
> also atomize

Atomize what? I asked you 4 times now, you never answered

**Review by TiviPlus — CHANGES_REQUESTED** [review]
A) the crates should be in the respective department
B) This comes forfree, change that

**Comment by BraveMole** [comment]
> A) the crates should be in the respective department

But why? 

> B) This comes forfree, change that

Nothing in this is added for free. Everything that was added is either free in a vendor, or taken from elsewhere. The exception is the poly pills bottle, which can be done for free in medbay

**Comment by TiviPlus** [comment]
> > A) the crates should be in the respective department
> 
> But why?
> 
> > B) This comes forfree, change that
> 
> Nothing in this is added for free. Everything that was added is either free in a vendor, or taken from elsewhere. The exception is the poly pills bottle, which can be done for free in medbay

Answered in discord

**Comment by MemeLauncher** [comment]
Just putting the stuff in req that goes to alamo roundstart is enough (not the ammo vendor, im talking about water bottle, MRE, flare, water/fuel tank, lasrifle battery charger), as this is the true pain of roundstart which could be us filling ammo boxes/making miner upgrades. And putting them outside on the hallway to alamo for marines is a gamble, sometimes they just don't bring it to alamo (worst case, destroyed there to feed roomba).

**Pull request closed without merging**